newapi/ooniapi/measurements.py

```diff
@@ -228,6 +239,26 @@ def _unwrap_post(post: dict) -> dict:
     raise Exception("Unexpected format")
 
 
+def _read_measurement_from_disk(msmt_uid: str) -> Optional[bytes]:
+    """Reads msmt from local disk cache or returns None"""
+    assert msmt_uid
+    if not msmt_uid.startswith("20"):
+        return None
```

Comment:
Isn't it better to assert this and raise, rather than returning a `None` value?

Reply:
It's not an exception because it's expected to happen often.